data/mods/Perk_melee/PERKS/insight.json

```
@@ -0,0 +1,71 @@
+[
+  {
+    "type": "mutation",
+    "id": "MELEE_PERK_INSIGHT",
+    "copy-from": "MELEE_PERK_BASE",
+    "name": { "str": "Insight I" },
+    "description": "You passively accumulate up to 20 insight stacks while near enemies.  Each insight stack increases all damage dealt by 1 until your next attack.  Insight greatly favors perceptive warriors.",
+    "flags": "MELEE_PERK_INSIGHT",
+    "enchantments": [ "melee_perk_ench_insight" ]
+  },
+  {
+    "type": "mutation",
+    "id": "MELEE_PERK_INSIGHT_DODGE",
+    "copy-from": "MELEE_PERK_BASE",
+    "name": { "str": "Like Water" },
+    "description": "You gain a +25 dodge chance if you have at least 10 insight stacks.",
+    "flags": "MELEE_PERK_LIKE_WATER",
+    "enchantments": [ "melee_perk_ench_insight_dodge" ]
+  },
+  {
+    "type": "mutation",
+    "id": "MELEE_PERK_SIXTH_SENSE",
+    "copy-from": "MELEE_PERK_BASE",
+    "name": { "str": "Sixth Sense" },
+    "description": "You accumulate up to 20 insight stacks even from enemies you can't see.  You can perceive unseen enemies within 10 tiles if you have at least 10 insight stacks.",
+    "flags": "MELEE_PERK_SIXTH_SENSE",
+    "enchantments": [ "melee_perk_ench_six_sense" ]
+  },
+  {
+    "type": "mutation",
+    "id": "MELEE_PERK_INSIGHT_INFINITE",
+    "copy-from": "MELEE_PERK_BASE",
+    "name": { "str": "The psychic sea" },
```

For consistency with other perk names.
```suggestion
    "name": { "str": "The Psychic Sea" },
```